apps/web/src/views/GaugesVoting/hooks/useUserVoteGauges.ts

```diff
@@ -35,7 +35,7 @@ export const useUserVoteSlopes = () => {
       userInfo?.cakePoolProxy,
       publicClient,
     ],
-
+    initialData: [],
```

Comment:
or use placeholderData

Reply:
yeah, my mistake

Reply:
Hi, I checked,  momoyil correct. 

Since set either initialData or placeholderData will suggest isloading to be false. 
Let's assume this is actually what should be the standard behavior. 
Introduct a memod state is to complex for this easy requirement.